attest/untrusted/src/lib.rs

```diff
@@ -82,10 +81,35 @@ impl QuotingEnclave {
     }
 }
 
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
+#[derive(Clone, Debug, Display, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub enum TargetInfoError {
+    /// SGX error: {0}
+    Sgx(SgxError),
+    /// Quote library error: {0}
+    Ql(mc_sgx_dcap_ql::Error),
+    /// Quoting enclave busy
+    QeBusy,
+    /// Error retrying: {0}
+    Retry(String),
+}
+
+impl From<mc_sgx_dcap_ql::Error> for TargetInfoError {
+    fn from(src: mc_sgx_dcap_ql::Error) -> Self {
+        match src {
+            mc_sgx_dcap_ql::Error::QuoteLibrary(QlError::Busy) => TargetInfoError::QeBusy,
+            e => TargetInfoError::Ql(e),
+        }
+    }
+}
```

Comment:
In particular the `mc-sgx-dcap-ql` crate is meant to be used in a `std` environment, it calls into functions that are available via `.so`s.
Because of this the `TargetInfoError` was brought out of the core types and moved here to the untrusted side.